src/devices/custom_devices_diy.ts

```diff
@@ -1167,6 +1168,53 @@ const definitions: Definition[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ['ESP32C6.PCIE-switch'],
+        model: 'ESP32C6.PCIE-switch',
+        vendor: 'Custom devices (DiY)',
+        description: 'PCIE switch',
+        fromZigbee: [
```

Comment:
Please move the `fromZigbee`/`toZigbee` converter to `tzLocal`/`fzLocal` (example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/adeo.ts#L11)